src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -203,6 +203,32 @@ public void newUserCreateEditMessageWallAndRenameDone() {
 
     Assertion.assertEquals(driver.getCurrentUrl(), expectedWallUrl);
     wall.verifyMessageEditTextRenameDone(title, message, newName);
+
+
+//
+//    wall.open(newName);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)